CruCentralCoast/Events/EventsVC.swift

```diff
@@ -48,47 +57,102 @@ class EventsTVC: UITableViewController {
             }
             self.tableView.reloadData()
         }
-        
-        let longPressGestureRecognizer = UILongPressGestureRecognizer(target: self, action: #selector(handleLongPressGesture(gestureRecognizer:)))
-        longPressGestureRecognizer.minimumPressDuration = 0.3
+        createLongPressGesture()
```

Comment:
Nice use of function decomposition rather than putting all of the code in one giant viewDidLoad method 👍

Reply:
`self.`